### PR TITLE
update set value description

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/data/api.tsx [hunk 1]
```diff
@@ -1521,12 +1634,38 @@ export default {
...
+              <p>
+                First time invoked and trigger form to be <code>dirty</code>
+              </p>
+            </li>
+            <li>
+              <p>
+                Update formState <code>touched</code>
+              </p>
+            </li>
+          </ul>
```
Comment (bluebill1049) [inline src/data/api.tsx]: @adhrinae can you please translate the above when you have time? feel free to commit into this branch.

File: src/data/api.tsx [hunk 2]
```diff
@@ -1535,15 +1674,38 @@ export default {
...
```
Comment (bluebill1049) [inline src/data/api.tsx]: @kotarella1110 can you please translate the above when you have time? feel free to commit into this branch.

File: src/data/api.tsx (hunk 1 above)
Comment (adhrinae) [reply src/data/api.tsx]: OK. I will.

Comment (bluebill1049) [reply src/data/api.tsx]: Thank you @adhrinae

**Comment by bluebill1049** [comment]
thank you @adhrinae

**Comment by bluebill1049** [comment]
@kotarella1110 I will merge this PR in, you can translate the Japanese version whenever you have time :) and thank you 👍

**Comment by kotarella1110** [comment]
@bluebill1049 OK! I will translate when i have time :muscle:

**Pull request merged**